core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/SimpleBufferedOutputStream.java

```diff
@@ -22,10 +22,13 @@
 import java.io.IOException;
 import java.io.OutputStream;
 
+import javax.annotation.concurrent.NotThreadSafe;
+
 /**
  * A simplified BufferedOutputStream with borrowed buffer, and allow users to see how much data have
  * been buffered.
  */
+@NotThreadSafe
 class SimpleBufferedOutputStream extends FilterOutputStream {
```

Comment:
The fact that this is a stream and the the impl itself show this is not a thread safe class